ngraph/python/mock_py_frontend/mock_py_ngraph_frontend/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET_FE_NAME "mock_py_ngraph_frontend")
```

Comment:
If we need to create `mock_py_ngraph_frontend` only for tests, can we move this library to test folder?

Reply:
unfortunately, existing 'ngraph/python/tests' folder is intended only for python scripts, there should not be such c++-based mock frontends. I can rename it to something like 'test_cpp' folder if needed

Reply:
Renamed mock_py_ngraph_frontend to 'test_cpp' for better consistence